napari/layers/image/image.py

```diff
@@ -1191,3 +1192,40 @@ def _calc_data_range(self, mode='data') -> Tuple[float, float]:
                 )
             )
         return calc_data_range(input_data, rgb=self.rgb)
+
+    def _raw_to_displayed(self, raw: np.ndarray) -> np.ndarray:
+        """Determine displayed image from raw image.
+
+        For normal image layers, just return the actual image.
+
+        Parameters
+        ----------
+        raw : array
+            Raw array.
+
+        Returns
+        -------
+        image : array
+            Displayed array.
+        """
+        lim_tup = _coerce_contrast_limits(self.contrast_limits)
```

Comment:
Could we call `lim_tup` something like `coerced_limits` ?

Reply:
I mean it's sort of obvious that lim_tup == limits_tuple, so maybe ignore.

Reply:
Or I guess could unpack the tuple into the limits tuple and the other two values.

Reply:
Or how horrible is just using:
`_coerce_contrast_limits(self.contrast_limits).contrast_limits`?
and then
`_coerce_contrast_limits(self.contrast_limits).scale`